src/Calculator/Controls/MathRichEditBox.cpp

```diff
@@ -178,6 +179,15 @@ void MathRichEditBox::BackSpace()
 
 void MathRichEditBox::SubmitEquation(EquationSubmissionSource source)
 {
+    // Clear formatting since the graph control doesn't work with bold/underlines
+    auto range = this->TextDocument->GetRange(0, this->TextDocument->Selection->EndPosition);
+
+    if (range != nullptr)
+    {
+        range->CharacterFormat->Bold = FormatEffect::Off;
```

Comment:
Can we clear all character formatting? Copy/Paste will keep the paragraph formatting (i.e. spacing above or below the text)